control-freec/11.5/Dockerfile

```diff
@@ -0,0 +1,27 @@
+################## BASE IMAGE ######################
+
+FROM biocontainers/biocontainers:latest
+
+################## METADATA ######################
+
+LABEL base_image="biocontainers:latest"
+LABEL version="1"
+LABEL software="Control-FREEC"
+LABEL software.version="11.5"
+LABEL about.summary="Prediction of copy numbers and allelic content using deep-sequencing data"
+LABEL about.home="http://boevalab.inf.ethz.ch/FREEC/index.html"
+LABEL about.documentation="http://boevalab.inf.ethz.ch/FREEC/index.html"
+LABEL extra.identifiers.biotools="FREEC"
```

Comment:
please use freec instead of FREEC for biotools entry (lowercase case is the real identifier in biotools), cf https://bio.tools/freec

Reply:
fixed!